.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -4,6 +4,7 @@ about: Suggest an idea for this project
 title: ''
 labels: ''
 assignees: ''
+
```

Comment:
Why?

Reply:
> Why?

Strange, I could've sworn that this was raised by mdl.

Or that at least it was the standard way to end the headers on YAML front
matter.

But I didn't find anything in the mdl rules:

* <https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md>

I see that this is currently done on `bug_report.md`:

* <https://raw.githubusercontent.com/netblue30/firejail/452d70cc77ddb308408c4d4da3a81a4b74530b65/.github/ISSUE_TEMPLATE/bug_report.md>

And also on Jekyll's (which invented YAML frontmatter AFAIK) issue templates:

* <https://raw.githubusercontent.com/jekyll/jekyll/v4.3.2/.github/ISSUE_TEMPLATE/documentation.md>
* <https://raw.githubusercontent.com/jekyll/jekyll/v4.3.2/.github/ISSUE_TEMPLATE/feature_request.md>

But not in the GitHub docs currently:

* <https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/manually-creating-a-single-issue-template-for-your-repository>

Maybe they changed it since last I checked or maybe this was a quirk in
Liquid/Jekyll or something similar.

Edit: I see that vim's syntax highlighting colors the last line before `---`
differently if there is no blank line before `---`, which might be related.